e2e/test/scenarios/embedding/embedding-dashboard.cy.spec.js

```diff
@@ -977,7 +987,25 @@ describeEE("scenarios > embedding > dashboard appearance", () => {
       );
     });
 
-    getIframeBody().findByText("Rows 1-21 of first 2000").should("exist");
+    getIframeBody().within(() => {
+      cy.findByText(questionDetails.name).should("exist");
+      cy.findByText("April 2022").should("exist");
+
+      // (metabase#49537)
+      // TODO: Enable these assertions once `echartsTooltip().should("be.visible")` fails when the tooltip is not visible
+      // chartPathWithFillColor("#509EE3").last().realHover();
+      // echartsTooltip().should("be.visible");
```

Comment:
This might be one of the rare cases we can't use `be.visible` since Cypress passes this assertion when the tooltip isn't visible.
![Screenshot 2024-11-14 at 5 19 26 PM](https://github.com/user-attachments/assets/7e75967a-75e6-443e-ab6d-78132ec4ad50)

I also tried [`Cypress.dom.isVisible()`](https://docs.cypress.io/api/cypress-api/dom#Is-visible) but it also doesn't fail this assertion. I suspect `should("be.visible")` uses `Cypress.dom.isVisible()` under the hood

I commented it out because this assertion still pass even when I revert the fix.